compiler/rustc_mir/src/interpret/terminator.rs

```diff
@@ -110,7 +110,7 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
             }
 
             Abort => {
-                M::abort(self)?;
+                M::abort(self, "aborted execution".to_owned())?;
```

Comment:
What message do we want here?

Reply:
The same as in the other place, I'd say.